SPECS/libmicrohttpd/libmicrohttpd.spec

```diff
@@ -52,24 +60,19 @@ Doxygen documentation for libmicrohttpd and some example source code
 %autosetup -p1
 
 %build
-# Required because patches modify .am files
-autoreconf --install --force
 %configure --disable-static --with-gnutls --enable-https=yes
 %make_build
-make -C doc/doxygen full
-
-# Disabled for now due to problems reported at
-# https://gnunet.org/bugs/view.php?id=1619
+# Using 'fast' to avoid BuildRequires on graphviz; when/if graphviz is available,
+# this can be switched to 'full' to build dependency graphs
+make -C doc/doxygen fast
 
 %check
-%ifnarch s390x
 %make_build check
-%endif
 
 %install
 %make_install
 
-find %{buildroot} -type f -name "*.la" -delete -print
+rm -f %{buildroot}%{_libdir}/*.la
```

Comment:
This is an outdated way of removing the `.la` files and we try to stick to the one from the previous version. It makes updates less prone to break during updates. Having said that, it's a minor detail, no strong pressure to revert.

Reply:
this is from our inherited spec file and it's better for us to use this format (i.e. the result is identical) so we reduce future delta.